### DISCUSSION TITLE
Unable Run the Simulation

### DISCUSSION THREAD

**Discussion by TsingYi11** [damask-multiphysics/DAMASK#25] [Q&A] [answered]
<<<+-  config init  -+>>>
 reading material configuration
Segmentation fault, 
After I rebuilt the EBSD file in DREAM3D software, configured geom, material, and load, and ran the simulation, an error message appeared. However, if I cut a part of the EBSD and repeated the same steps, I found that it could run, but the result differed from what I wanted, and the strain value was minimal. I hope to get your help.

**Comment by MarDiehl** [comment, marked as answer]
probably you run out of memory.

**Reply by TsingYi11** [reply]
Thanks

**Reply by shiningpy** [reply]
Hi, there, do you guys figure this issue out? I encounter the same problem with yours. I tried almost everything to reconfig the material.yaml and goem.vti from the dream3D file.  However, the propramme keep reporting: 
<<<+-  config init  -+>>>
reading material configuration
Segmentation fault (core dumped)
Could you please help me out of this?

**Reply by MarDiehl** [reply]
without further details it's not possible to give an answer. pls have a look at https://damask-multiphysics.org/support/helpdesk

**Reply by shiningpy** [reply]
Thank you very much for your response.

**Reply by shiningpy** [reply]
> without further details it's not possible to give an answer. pls have a look at https://damask-multiphysics.org/support/helpdesk
I have one more question. Can DAMASK be used to simulate the deformation behavior of NiTi alloys? NiTi alloys involve a stress-induced martensitic transformation, which causes a phase change from cubic B2 to monoclinic B19'.
